demos/demo_tricamera.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""
+This demo is to start three pylon dependent cameras, view their image streams,
+and if desired, store the timestamps from them to analyse how well the three
+cameras are in sync with each other.
+"""
+import argparse
+import numpy as np
+import cv2
+import pickle
+
+import robot_interfaces
+
+
+def main():
+    argparser = argparse.ArgumentParser(description=__doc__)
+    arg_action_group = argparser.add_mutually_exclusive_group(required=False)
```

Comment:
There is no need for a mutually exclusive group if there is only one argument anyway.  You can use `argparser.add_argument`